DESCRIPTION

```diff
@@ -27,7 +31,11 @@ Imports:
     cluster,
     grid,
     igraph,
-    VennDiagram
+    VennDiagram,
+    GOSemSim,
+    ggcorrplot,
+    ggplot2,
+    org.Hs.eg.db
```

Comment:
**We won't merge that in.** I'll comment on the mailing list. You are not the only one to try this.

Reply:
I have added new functions and files that will not merge org.Hs.eg.db